setup.py

```diff
@@ -1,15 +1,28 @@
-from setuptools import setup, find_packages
+from setuptools import setup
 
 setup(
-    name = "libkeepass",
-    version = "0.1.2",
-    packages = ["libkeepass"],
-    author = "Lukas Koell",
-    author_email = "phpwutz@gmail.com",
-    description = "A library to access keepass v3 (keepass1) and v4 (keepass2) files",
-    license = "GPL",
-    keywords = "keepass library",
-    url = "https://github.com/phpwutz/libkeepass",   # project home page, if any
-    install_requires=["lxml==3.2.1", "nose==1.3.0", "pycrypto==2.6.1"],
+    name="libkeepass",
+    version="0.1.2",
+    packages=["libkeepass"],
+    author="Lukas Koell",
+    author_email="phpwutz@gmail.com",
+    description="A library to access KeePass 1.x/KeePassX (v3) and KeePass "
+                "2.x (v4) files",
+    license="GPL",
+    keywords="keepass library",
+    url="https://github.com/phpwutz/libkeepass",  # project home page, if any
+    test_suite="tests",
+    install_requires=[
+        "lxml>=3.2.1",
+        "nose>=1.3.0",
+        "pycrypto>=2.6.1",
+        "colorama>=0.3.2"
+    ],
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.2",
+        "Programming Language :: Python :: 3.3",
+    ],
     test_suite="tests"
```

Comment:
Lines 27 and 14 are the same. This leads to the following error when trying to install using pip:
```Processing /home/pschmitt/dev/libkeepass
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-fOzaVK-build/setup.py", line 27
        test_suite="tests"
    SyntaxError: keyword argument repeated

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-fOzaVK-build/
```